Gems/Atom/Feature/Common/Code/CMakeLists.txt

```diff
@@ -9,48 +9,44 @@
 o3de_pal_dir(pal_source_dir ${CMAKE_CURRENT_LIST_DIR}/Source/Platform/${PAL_PLATFORM_NAME} "${gem_restricted_path}" "${gem_path}" "${gem_parent_relative_path}")
 set(common_source_dir ${CMAKE_CURRENT_LIST_DIR}/Source/Platform/Common)
 
+# TODO(OBJECT-library): Change this target to an object library after Gradle was updated
 ly_add_target(
-    NAME ${gem_name}.Static STATIC
+    NAME ${gem_name}.Private.Object STATIC
     NAMESPACE Gem
     FILES_CMAKE
         atom_feature_common_files.cmake
-        atom_feature_common_staticlibrary_files.cmake
         ${pal_source_dir}/platform_${PAL_PLATFORM_NAME_LOWERCASE}_files.cmake
     PLATFORM_INCLUDE_FILES
         ${pal_source_dir}/platform_${PAL_PLATFORM_NAME_LOWERCASE}.cmake
     INCLUDE_DIRECTORIES
         PRIVATE
-            .
-            Source
-            ${pal_source_dir}
-        PUBLIC
             Include
             Source
+            ${pal_source_dir}
+            3rdParty/ACES
```

Comment:
No, this is a include directory (`Gems/Atom/Feature/Common/Code/3rdParty/ACES`) where the source of ACES (it's just a header and a cpp) resides. ACES has been there since c2cbd430fe304de447ca81df7625f447d0ed88ba